apps/fluent-tester/src/TestComponents/Button/ButtonSizeTestSection.tsx

```diff
@@ -61,7 +61,7 @@ export const ButtonSizeTest: React.FunctionComponent = () => {
       <Button size="large" style={commonTestStyles.vmargin}>
         Large
       </Button>
-      {Platform.OS !== 'android' && (
+      {Platform.OS !== 'android' && Platform.OS !== 'ios' && (
```

Comment:
nit: Can we move this to a 'isMobile' variable?

Reply:
What do you think about making an isMobile method in @fluentui-react-native/framework? I was thinking I could do that and update all the references to android/ios platform checks then

Reply:
I answered this elsewhere, but I don't like that mainly because iPad is weird. Sometimes we want to filter if we iPad or not. For that reason, I've preferred just doing the checks individually

Reply:
I'm not super against it, but it's a small enough thing that I wasn't sure it was necessary to add a new package for it / add it to an existing package

Reply:
Actually with Mike's recent work with trait collections, that might be worth revising once that is upstreamed

Reply:
We could have separate isMobile/isMobileSmallScreen checks? isMobile would always include iPad, isMobileSmallScreen would only include ipad compact mode. 

@mischreiber do you have any preferences?

Reply:
To further elaborate my thought: an `isMobile` check could include the horizontal size class (i.e: `Platform.OS === 'iOS' && sizeClass === 'compact')`. But it's still context dependent whether we care about the size class. In this case, I don't think we'd want the compound button even on iPad.


I'd prefer that any "consolidation" of platform checks happen as a separate PR. I feel like it's something we get in a loop about and I'd rather it not block a PR.

Reply:
Yep I'll bring it up in the next sync, will leave this PR as is